contracts/WalletSimple.sol

```diff
@@ -553,7 +543,7 @@ contract WalletSimple is IERC721Receiver, ERC1155Receiver {
    * greater than the minimum element in the window.
    * @param sequenceId to insert into array of stored ids
    */
-  function tryInsertSequenceId(uint256 sequenceId) private onlySigner {
+  function tryInsertSequenceId(uint256 sequenceId) private {
```

Comment:
Private means can only be called internally?

Reply:
Yes, Private functions can only be called from inside the contract, even the inherited contracts can’t call them.